src/OpenTelemetry.Instrumentation.ConfluentKafka/README.md

```diff
@@ -9,8 +9,124 @@
 [![NuGet download count badge](https://img.shields.io/nuget/dt/OpenTelemetry.Instrumentation.ConfluentKafka)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.ConfluentKafka)
 [![codecov.io](https://codecov.io/gh/open-telemetry/opentelemetry-dotnet-contrib/branch/main/graphs/badge.svg?flag=unittests-Instrumentation.ConfluentKafka)](https://app.codecov.io/gh/open-telemetry/opentelemetry-dotnet-contrib?flags[0]=unittests-Instrumentation.ConfluentKafka)
 
-Download the `OpenTelemetry.Instrumentation.ConfluentKafka` package:
+## Usage
```

Comment:
@g7ed6e could you point me to a doc/update this(later PR(when you have time)) on what metrics/which traces are published ? It would be great for community.